boot-core/examples/cw20.rs

```diff
@@ -1,26 +1,25 @@
-use boot_core::networks::juno::LOCAL_JUNO;
+use boot_core::networks::LOCAL_JUNO;
 
-use boot_core::instantiate_daemon_env;
+use boot_core::prelude::*;
```

Comment:
This is minor but would we be able to we avoid using glob imports? It makes the code harder to read because you don't necessarily know what's coming from where.

Reply:
Yeah I agree. That's why I use a separate prelude module. The reason is that a user needs to have the BootExecute, BootInstantiate, ... traits in scope before they can access the function calls on the contracts like `contract.execute()`.

So `prelude` only imports those specific traits to just make them available.